zwave-classifier.js

```diff
@@ -70,6 +72,30 @@ const NOTIFICATION_SENSOR = {
     },
     valueListMap: [false, true],
   },
+  [NOTIFICATION_CO_DETECTOR]: {// 2
+    name: 'carbon_monoxide',
+    propertyName: 'carbon_monoxide',
+    propertyDescr: {
+      '@type': 'CarbonMonoxideProperty',
```

Comment:
We don't yet have a CarbonMonoxideProperty, so this should be AlarmProperty for now.

The @type for the node should also be:
```
    '@type': ['Alarm'],
```

I wound up adding an entry for CO already, so it would be great if you could rebase this against the latest master.

I also added a json file for the classifier here: https://github.com/mozilla-iot/zwave-adapter/blob/master/test/classifier/Fibaro-FGSD001-CO-Sensor.json

It will need to have some extra properties added to it in order for the tests to pass.

Reply:
Hi @dhylands, how do I run the tests locally? I tried `npm test` but it did not work

Reply:
The zwValues fields reflect what was actually retrieved from the device. So the min/max being 0 for 3-49-1-40 means that's what the device reported.

Reply:
The `zwValues` are what the device reports. The `properties` are the properties that the classifier creates.

What the test is doing is basically using zwValues as inputs and then generating a set of properties and comparing that the generated properties match the properties in the .json file.

So if the classifier adds new properties, then the .json file needs to be updated to include the new properties.